lstchain/scripts/lstchain_dl1_to_dl2.py

```diff
@@ -123,11 +142,13 @@ def main():
     dl1_keys = get_dataset_keys(args.input_file)
     if dl1_images_lstcam_key in dl1_keys:
         dl1_keys.remove(dl1_images_lstcam_key)
+    
     if dl1_params_lstcam_key in dl1_keys:
         dl1_keys.remove(dl1_params_lstcam_key)
 
-    if dl1_params_src_dep_lstcam_key in dl1_keys:
-        dl1_keys.remove(dl1_params_src_dep_lstcam_key)
+    for i in reversed(range(len(dl1_keys))):
```

Comment:
Why is this construct needed? Can't you just do
```
for i in dl1_keys:
    ....
```

Reply:
It was needed because an index could be changed while the loop removing the elements.
but anyway I removed this part, and now it is simple